docs/components/customization/localization.mdx

```diff
@@ -73,9 +74,22 @@ function MyApp({ Component, pageProps }: AppProps) {
 export default MyApp;
 ```
 
+## Adding or updating a localization 
+
+Our localizations are customer sourced and we encourage customers to add or update localizations. The process is:
+
+1. Fork the [https://github.com/clerkinc/javascript/](https://github.com/clerkinc/javascript/) repo.
+2. Clone it locally to edit it.
+3. Review our [Contributing](https://github.com/clerkinc/javascript/blob/main/docs/CONTRIBUTING.md) guide.
+4. If you are updating an existing localization locate the file in `packages/localizations/src`
+5. If you are adding a new language copy the en-US.ts file and name it according to your language. The naming is the abbreciated language-region. For French in Canada it would be fr-CA.ts.
```

Comment:
```suggestion
5. If you are adding a new language, copy the `en-US.ts` file and name it according to your language. The naming is the abbreviated language-region. For example, for French in Canada, it would be `fr-CA.ts.`
```